backend/question-service/src/dto/question.dto.ts

```diff
@@ -37,6 +46,7 @@ export class FilterQuestionsDto {
 
     @ApiProperty()
     @IsArray()
+    @ArrayNotEmpty()
     @IsOptional()
```

Comment:
Mostly ok, but this part seems contradictory to me. It works fine if empty?

Reply:
Technically it's more of we allow topics to not be part of the query parameters (IsOptional), but if it is then it should not be empty (ArrayNotEmpty) otherwise we won't get any results anyways. But then again, it's not possible (I think) to enter an empty array in query parameters so we don't really need it.

Reply:
Ah got it!